linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_checking_systemd_timer/tests/aide_timer_weekly.pass.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# packages = aide,systemd
```

Comment:
I wonder if `systemd` is indeed necessary here?

Reply:
well my idea was we cannot define a systemd timer without the systemd, or you mean majority systems are with systemd. I also thought of that, but still from my experience there still deployed openrc or similar based distros, so figured better be explicit about it :)

Reply:
All products currently supported in the project use `systemd`:
`grep -r "init_system:" products/`

It is not a problem to have this package in the test scenario. I was only considering its efficiency. It is one package more to check for each test scenario script. If it is not necessary, we can save some seconds.